packages/inquirer_oc/inquirer_oc.1.0.14/opam

```diff
@@ -0,0 +1,56 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis: "A collection of common interactive command line user interfaces"
+description:
+  "A collection of common interactive command line user interfaces"
+maintainer: ["Ou Fumio"]
+authors: ["Ou Fumio"]
+license: "MIT"
+tags: [
+  "answer"
+  "answers"
+  "ask"
+  "base"
+  "cli"
+  "command"
+  "command-line"
+  "confirm"
+  "enquirer"
+  "generate"
+  "generator"
+  "hyper"
+  "input"
+  "inquire"
+  "inquirer"
+]
+homepage: "https://github.com/araera111/inquirer_oc"
+doc: "https://github.com/araera111/inquirer_oc/#README"
+bug-reports: "https://github.com/araera111/inquirer_oc/issues"
+depends: [
+  "ocaml"
+  "dune" {>= "3.7"}
+  "odoc" {with-doc}
+]
+build: [
+  ["dune" "subst"] {dev}
+  [
+    "dune"
+    "build"
+    "-p"
+    name
+    "-j"
+    jobs
+    "@install"
+    "@runtest" {with-test}
+    "@doc" {with-doc}
+  ]
+]
+dev-repo: "git+https://github.com/araera111/inquirer_oc.git"
+url {
+  src:
+    "https://github.com/araera111/inquirer_oc/archive/refs/tags/v1.0.14.tar.gz"
```

Comment:
This archive still contains the `_build` directory

Reply:
Sorry. I deleted _build and re-uploaded the tag. I have confirmed that _build is not present in the .zip.